apps/api/middleware/jwt_auth.py

```
@@ -45,20 +45,63 @@ class AuthClaims(TypedDict, total=False):
     aud: NotRequired[str | list[str]]
 
 
-AUTH_SECRET_ENV_VAR = "AUTH_SECRET"
+AUTH_SECRET_ENV_VAR: Final[str] = "AUTH_SECRET"
```

Ruff (0.14.1)

Possible hardcoded password assigned to: "AUTH_SECRET_ENV_VAR"

(S105)